server/server/tensorboard.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask
+from loguru import logger
+from tensorboard.program import TensorBoard
+from werkzeug.serving import is_running_from_reloader
+
+from server.interface import Interface
+
+HOST = "0.0.0.0"
+DEFAULT_TENSORBOARD_PORT = "6006"
+RUNNING = "tensorboard_running"
+
+
+def setup_tensorboard(app: Flask) -> None:
```

Comment:
nit: `setup_tensorboard` doesn't really convey that it's also launching the app.

Reply:
Changed to `launch_x` in latest revision